### PR TITLE
Updated taxibot.png

### PR DESCRIPTION
**Pull request by Diebeck**
## About the PR

Updates the taxibot textures to be newer and fancier.

**Media**

- [X ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- tweak: Updated Taxibot sprites

### CODE REVIEW

**Comment by Emisse** [comment]
update the meta.json to include that you tweaked the sprites

**Comment by Diebeck** [comment]
Ive done it now, anything else to do?

**Comment by Flareguy** [comment]
Please add images showing this sprite in-game.

File: Resources/Textures/Mobs/Silicon/Bots/taxibot.rsi/meta.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
     "y": 32
   },
   "license": "CC-BY-SA-3.0",
-  "copyright": "taxibot drawn by FillerVK",
+  "copyright": "taxibot drawn by Diebeck",
```
Comment (ficcialfaint) [inline Resources/Textures/Mobs/Silicon/Bots/taxibot.rsi/meta.json:8]:
```suggestion
  "copyright": "taxibot drawn by FillerVK, modified by Diebeck",
```

**Review by ficcialfaint — CHANGES_REQUESTED** [review]

**Comment by FillerVK** [comment]
It looks good, but there are a few comments.
- Could you please make this "eye" the way it was before, it just doesn't fit in with the rest of the bots.
- TaxiBot looks like a jeep from the side, not a small taxi robot
- I don't think it's cool that you made it more "square"
- Maybe reduce the headlights and leave the taxi markings?

Thank you if you do this!

**Comment by EmoGarbage404** [comment]
@Diebeck is this pr still being worked on?

**Comment by AJCM-git** [comment]
re-open once you come back

**Pull request closed without merging**